### PR TITLE
Fix #10952: Stop & disable playback when no parts are visible

### LINKED ISSUE #10952: [MU4 Issue] Playback cursor should not be visible when all instruments hidden in the score
**Describe the bug**
See #9357 and https://github.com/musescore/MuseScore/issues/9357#issuecomment-1082265083

https://github.com/musescore/MuseScore/assets/90187801/ace24e80-5289-40d4-a87d-f17fe7e3db41

**Platform information**
macOS, Windows, Linux

### PR DESCRIPTION
**Pull request by mathesoncalum**
Resolves: #10952

### CODE REVIEW

**Comment by DmitryArefiev** [comment]
Tested #10952 on Win10, Mac14, LinuxUbuntu 22.04 - **FIXED**

File: src/notation/view/playbackcursor.cpp [hunk 1]
```diff
@@ -137,7 +137,11 @@ bool PlaybackCursor::visible() const
 
 void PlaybackCursor::setVisible(bool arg)
 {
-    m_visible = arg;
```
Comment (RomanPudashkin) [inline src/notation/view/playbackcursor.cpp:140]: The previous implementation was better :) This method should do exactly what its name says. It's better to control the visibility outside this class

**Comment by bkunda** [comment]
This certainly improves things, although my initial thought was that playback should simply be stopped when there are no visible instruments in the score (and as @Tantacrul suggests [here](https://github.com/musescore/MuseScore/issues/9357#issuecomment-932899372), including that the playback panel then would become deactivated).
But it’s really quite trivial: the case of a user actually wanting to hide every instrument in their score is extremely marginal at best. It’s really a nonsensical use case, so the current behaviour at least doesn’t appear buggy anymore.

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
@mathesoncalum All good, but can we save the playback cursor position when stop playback?


https://github.com/musescore/MuseScore/assets/90187801/22b5a096-e837-4dd4-82cd-64688932f1f0

**Comment by mathesoncalum** [comment]
> save the playback cursor position...

@DmitryArefiev of course! Done so in latest push.

**Comment by DmitryArefiev** [comment]
@mathesoncalum Yeah, it works fine now. Thanks!

Tested #10952 on Win10, Mac13.6 - **FIXED**

**Pull request merged**